core/src/client/async_client/mod.rs

```diff
@@ -130,9 +136,31 @@ impl ClientBuilder {
 			id_manager: RequestIdManager::new(self.max_concurrent_requests, self.id_kind),
 		}
 	}
+
+	/// Build the client with given transport.
+	#[cfg(feature = "async-wasm-client")]
+	pub fn build_with_wasm<S, R>(self, sender: S, receiver: R) -> Client
```

Comment:
`build_for_wasm`/`build_for_tokio`?

Reply:
Another option is to keep a single `build` method and then use `cfg_if` to conditionally compile just the spawning?

Reply:
The reason I did like this is to avoid to have `mutually exclusive features` for `async-client` and `async-wasm-client` this way it becomes much cleaner and it should rather `intuitive` because one have to explicit call `build_with_wasm/build_with_tokio`..

opinions?

Reply:
You're right, this is better.